config/keybindings.go

```diff
@@ -1,24 +1,216 @@
 package config
 
 import (
-	"errors"
+	"strings"
 
+	"github.com/gdamore/tcell"
 	"github.com/spf13/viper"
 )
 
-// KeyToNum returns the number on the user's keyboard they pressed,
-// using the rune returned when when they press Shift+Num.
-// The error is not nil if the provided key is invalid.
-func KeyToNum(key rune) (int, error) {
-	runes := []rune(viper.GetString("keybindings.shift_numbers"))
-	for i := range runes {
-		if key == runes[i] {
-			if i == len(runes)-1 {
-				// Last key is 0, not 10
-				return 0, nil
+const (
+	CmdInvalid = 0
+	CmdLink1   = 1
+	CmdLink2   = 2
+	CmdLink3   = 3
+	CmdLink4   = 4
+	CmdLink5   = 5
+	CmdLink6   = 6
+	CmdLink7   = 7
+	CmdLink8   = 8
+	CmdLink9   = 9
+	CmdLink0   = 10
+	CmdTab1    = 11
+	CmdTab2    = 12
+	CmdTab3    = 13
+	CmdTab4    = 14
+	CmdTab5    = 15
+	CmdTab6    = 16
+	CmdTab7    = 17
+	CmdTab8    = 18
+	CmdTab9    = 19
+	CmdTab0    = 20
+	CmdBottom  = iota
```

Comment:
Is there any reason why you did it like this? You can just specify `= iota` at the start.

Reply:
It was meant as a semantic warning: there's some code that depends on these values being in this relation to each other.  Essentially: CmdTab0 - CmdTab1 == 9, and CmdTab[N+1] == 1 + CmdTabN, thus rearranging these would break stuff.  Basically, get future programmers to ask, wait, why does this break convention?

I'll add a comment to this effect.

Reply:
Alright, that makes sense. And yeah a comment is helpful, thanks.